docs/source/migrations.rst

```diff
@@ -52,14 +52,32 @@ Version 2.7.1
 
 - There are no special migrations needed for this version. Simply run `./manage.py migrate`.
 
+**Important Note:**
+
+If upgrading from < 2.7.0 to >= 2.7.1 or greater you may encounter a failed migration with ``0118_match_merge_link_all_orgs``.  Run the following code manually to fully migrate:
```

Comment:
@adrian-lara or @axelstudios -- can you provide a bit more background on why this is failing for the user. I just want to make sure that the user realizes that their data are still okay.